tests/test_brew_collector.py

```diff
@@ -116,8 +117,9 @@ def test_get_component_data(build_id, build_source, build_ns, build_name, licens
     c = Brew().get_component_data(build_id)
     if build_type == "module":
         assert list(c.keys()) == ["type", "namespace", "meta", "analysis_meta", "build_meta"]
-    elif build_type == "maven":
-        assert list(c.keys()) == ["type", "namespace", "meta", "build_meta"]
+    # TODO: uncomment when Maven build type is supported
+    # elif build_type == "maven":
+    #     assert list(c.keys()) == ["type", "namespace", "meta", "build_meta"]
```

Comment:
I'd prefer using ```with pytest.raises(BrewBuildTypeNotSupported)``` here.

Reply:
We're not calling this though (and no unsupported build types are present in test parameters), and whenever support is added, it won't raise the exception.

Reply:
I meant uncommenting the Maven-type test data defined in build_corpus above, then explicitly testing we raise the right exception for unsupported types. But it's fine to leave it commented I guess.

Reply:
I'll leave it as is for now.